docs/axelar/intro-to-axelar.md

```diff
@@ -0,0 +1,103 @@
+# Axelar Integration
+
+The Axelar-XRPL integration enables messages to pass between the XRPL and other Axelar-supported blockchains. This integration enables the transfer of XRP and XRPL-issued tokens to other chains and back; this also works for briding non-XRPL tokens from their native chains to XRPL.
```

Comment:
suggest changing to "XRPL and other blockchains that are connected through the Axelar network"

Reply:
1. Suggest a change to "transfer of XRPL- issued tokens from the XRPL to other chains and the transfer of tokens on other chains (such as ERC-20 tokens on EVM compatible chains) to the XRPL"
2. The asset XRP won't go through the Axelar bridge.

Reply:
1. Suggest a change to "transfer of XRPL- issued tokens from the XRPL to other chains and the transfer of tokens on other chains (such as ERC-20 tokens on EVM compatible chains) to the XRPL"
2. The asset XRP won't go through the Axelar bridge. 
3. typo in 'briding' -> change to 'bridging'